nengo/tests/test_copy.py

```diff
@@ -0,0 +1,228 @@
+from copy import copy
+import warnings
+
+import numpy as np
+import pytest
+
+import nengo
+from nengo.exceptions import NetworkContextError, NotAddedToNetworkWarning
+from nengo.params import params
+from nengo.utils.compat import is_array_like, pickle
+from nengo.utils.testing import warns
+
+
+@pytest.fixture(scope='module', autouse=True)
+def error_on_not_added_to_network_warning(request):
+    catcher = warnings.catch_warnings(record=True)
+    catcher.__enter__()
+    warnings.simplefilter(
+        'error', category=NotAddedToNetworkWarning)
+    request.addfinalizer(lambda: catcher.__exit__(None, None, None))
+
+
+def assert_is_copy(cp, original):
```

Comment:
This might just be me being a noob at `pytest`, but how are these parameters defined? I thought they had to be `pytest` fixtures? I see that `original` is paramterized at the bottom of the file.